CommandLine/testing/SimpleTests/data_structure_test.sog/create.edl

```diff
@@ -24,11 +24,21 @@ gtest_assert_eq(ds_list_find_value(list_str, 0), "alarm");
 gtest_assert_eq(ds_list_find_value(list_str, 1), "bank");
 gtest_assert_eq(ds_list_find_value(list_str, 2), "cash");
 
-random_set_seed(65079);
-ds_list_shuffle(list_str);
-gtest_assert_eq(ds_list_find_value(list_str, 0), "bank");
-gtest_assert_eq(ds_list_find_value(list_str, 1), "alarm");
-gtest_assert_eq(ds_list_find_value(list_str, 2), "cash");
+// When shuffle is repeated with the same seed, it results
+// in a repeated set of orderings modulo 2.
+for (var i = 0; i < 5; ++i) {
+	random_set_seed(69378);
+	ds_list_shuffle(list_str);
+	if (i % 2) {
+		gtest_assert_eq(ds_list_find_value(list_str, 0), "alarm");
+		gtest_assert_eq(ds_list_find_value(list_str, 1), "bank");
+		gtest_assert_eq(ds_list_find_value(list_str, 2), "cash");
+	} else {
+		gtest_assert_eq(ds_list_find_value(list_str, 0), "bank");
+		gtest_assert_eq(ds_list_find_value(list_str, 1), "alarm");
+		gtest_assert_eq(ds_list_find_value(list_str, 2), "cash");
+	}
+}
```

Comment:
HA, I tested this in GM8.1 and apparently ENIGMA is just slightly different than GM8 here. Not only does GM8.1 repeat a different sequence, it repeats the sequence modulo 3 instead of modulo 2 when repeatedly given the same seed.

```cpp
list_str = ds_list_create();
list_num = ds_list_create();

ds_list_add(list_str, "alarm");
ds_list_add(list_str, "bank");
ds_list_add(list_str, "cash");

ds_list_sort(list_str, false);

ds_list_sort(list_str, true);

// When shuffle is repeated with the same seed, it results
// in a repeated set of orderings modulo 2.
for (i = 0; i < 5; i+=1) {
    random_set_seed(69378);
    ds_list_shuffle(list_str);
    show_message(ds_list_find_value(list_str, 0) + " " + ds_list_find_value(list_str, 1) + " " + ds_list_find_value(list_str, 2));
}
```